### PR TITLE
Kartikey_Assignment_6_Done

### PR DESCRIPTION
**Pull request by KartikeyDJune20**
image

### CODE REVIEW

File: src/kartikey/WebTableEmployeeManager.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		/*LinkedHashMap<String, Integer>lhm=new LinkedHashMap<String, Integer>();
+		for(WebElement element:listOfManagerID) {
+			if(lhm.containsKey(element.getText())) 
+				lhm.put(element.getText(), lhm.get(element.getText())+1);
+			else
+				lhm.put(element.getText(), 1);		
+		}
+		System.out.println(lhm);*/
+//************************************<By Set>***********************************		
+		HashSet<String> hs= new HashSet<String>();
```
Comment (harshvegada) [inline src/kartikey/WebTableEmployeeManager.java:43]: please give variable name correctly so third person can understand what that set do

File: src/kartikey/WebTableEmployeeManager.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		for(WebElement element:listOfManagerID) {
+			hs.add(element.getText());
+		}
+		//System.out.println(hs);		
+		int maxEmpManager=0;
+		String maxEmpManagerID="";
+		for(String str:hs) {
```
Comment (harshvegada) [inline src/kartikey/WebTableEmployeeManager.java:50]: what is purpose of writing two loops

Comment (harshvegada) [reply src/kartikey/WebTableEmployeeManager.java:50]: hs has unique emplist & listOfManagerID have all details
then why we need to check two times
please update code

**Review by harshvegada — COMMENTED** [review]
update, code

File: src/kartikey/WebTableEmployeeManager.java (hunk 1 above)
Comment (KartikeyDJune20) [reply src/kartikey/WebTableEmployeeManager.java:43]: Yes, my bad updated..

File: src/kartikey/WebTableEmployeeManager.java (hunk 2 above)
Comment (KartikeyDJune20) [reply src/kartikey/WebTableEmployeeManager.java:50]: Hello Harsh, Second loop is to get the maximum no. of employees that reports to manager, From first i get the Unique Managers and from the second i get how many employees are reporting to that particular manager.
I tried to fit in one loop but i m stuck as how to add+iterate+than re check to give count.. i need help will contact you tomorrow

**Pull request merged**